packages/powersync/lib/src/powersync_database.dart

```diff
@@ -602,3 +643,15 @@ Future<void> _powerSyncDatabaseIsolate(
     throw error;
   });
 }
+
+enum LogType {
```

Comment:
this should probably be in [packages/powersync/lib/src/log.dart](https://github.com/powersync-ja/powersync.dart/pull/53/files#diff-91cae18e041640056785a3479792bd713b26429c83ecce1bc1236a8474018288)